packages/button/src/Button.story.tsx

```diff
@@ -1,24 +1,53 @@
+/* eslint-disable react/jsx-key */
 /* eslint-disable react/display-name */
 import React from 'react';
-import { StoryFn } from '@storybook/react';
 
 import Icon, { glyphs } from '@leafygreen-ui/icon';
-import { storybookArgTypes, type StoryMetaType } from '@leafygreen-ui/lib';
+import {
+  storybookArgTypes,
+  type StoryMetaType,
+  type StoryType,
+} from '@leafygreen-ui/lib';
 
 import { Size } from './types';
 import Button, { ButtonProps, Variant } from '.';
 
 const meta: StoryMetaType<typeof Button> = {
   title: 'Components/Button',
   component: Button,
+
+  parameters: {
+    default: 'LiveExample',
+    generate: {
```

Comment:
Can we only run this if there's an environment variable set to Chromatic?

Reply:
Technically yes, but why?

Reply:
AFAIK there's no significant performance bottleneck for adding more stories to the sidebar.

Also, IMO, it's good to have these tests available while developing so we can quickly see if the changes we've made are correct

Reply:
@TheSonOfThomp Verify this